modules/nixos/default.nix

```
@@ -226,8 +226,9 @@ in {
       mkIf (cfg.linker != null)
       {
         systemd.services.hjem-activate = {
-          requiredBy = ["sysinit-reactivation.target"];
-          before = ["sysinit-reactivation.target"];
+          after = ["local-fs.target"];
+          requiredBy = ["sysinit-reactivation.target" "multi-user.target"];
```

IMO this dependency should be weakened

```suggestion
          wantedBy = ["sysinit-reactivation.target" "multi-user.target"];
```